includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -2190,6 +2190,7 @@ public function add_bnpl_debug_metadata( $metadata, $order ) {
 			[
 				'is_legacy_checkout_enabled' => ! WC_Stripe_Feature_Flags::is_upe_checkout_enabled(),
 				'is_oc_enabled'              => $this->is_oc_enabled(),
+				'pmc_enabled'                => 'yes' === $this->get_option( 'pmc_enabled' ),
```

Comment:
Do we want to capture more possible states for this value? It feels like it would help to know if the value is disabled vs non initialised, as an example.

Reply:
That's a good point. Updated it in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4650/commits/9b71b6eb8447c23979da927587b1d6c157581738